src/main/java/net/dv8tion/jda/internal/entities/StageChannelImpl.java

```diff
@@ -197,4 +197,12 @@ public StageChannelImpl setStageInstance(StageInstance instance)
         this.instance = instance;
         return this;
     }
+
+    // -- Object Overrides --
+
+    @Override
+    public String toString()
+    {
+        return "StaC:" + getName() + '(' + getId() + ')';
```

Comment:
I'm beginning to wonder if we should lose the weird abbreviation system we have and instead do `StageChannel` or `STAGE` (which models after ChannelType) to better represent channels in `toString` instead of `SC`, `StaC`, or whatever.
@MinnDevelopment Is there a strong reason _not_ to go a bit more explicit?

Reply:
No, i would also prefer a longer toString that shows the full type